backend/curation/api/v1/curation/collections/collection_id/s3_upload_credentials.py

```diff
@@ -29,6 +30,9 @@ def get(collection_id: str, token_info: dict):
         upload_key_prefix = f"super/{collection_id}/"
     else:
         upload_key_prefix = f"{user_info.user_id}/{collection_id}/"
+    rdev_prefix = os.environ.get("REMOTE_DEV_PREFIX", "").strip("/")
+    if rdev_prefix:
+        upload_key_prefix = f"{rdev_prefix}/{upload_key_prefix}"
```

Comment:
see comment above. Remote dev prefix is part of the _key_, not the _bucket_.

Reply:
It was not noticed. It would cause an issue only in rdev, and only in certain scenarios. But it was wrong and should be fixed.